docs/dev/java_lambdas.md

```diff
@@ -135,4 +135,4 @@ public static class DoubleTuple extends Tuple2<Integer, Integer> {
         this.f1 = f1;
     }
 }
-{% endhighlight %}
\ No newline at end of file
+{% endhighlight %}
```

Comment:
could you revert this change? It's probably just an automatically-added line-ending.

Reply:
Done. Thanks for the review!